.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+# This is a basic workflow that is manually triggered
+
+name: compile osmosis
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on: [push, pull_request, workflow_dispatch]
+
+# This workflow makes x86_64 binaries for mac, windows, and linux.
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+        targetos: [darwin, linux]
+        include:
+          - targetos: windows
+            arch: amd64
+    name: dig ${{ matrix.arch }} for ${{ matrix.targetos }}
```

Comment:
Why is it called dig? I thought dig is a DNS utility

Reply:
oops!

That is taken from another chain I am working on called dig, and yes, its binary gets confused with the dns tool often. 

Thanks!

Reply:
ah lol, makes sense now! Name collisions are the worst